doxygen/layout_templates/base_struct.xml

```diff
@@ -0,0 +1,275 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<doxygenlayout version="1.0">
+  <!-- Generated by doxygen 1.9.8 -->
+  <!-- Navigation index tabs for HTML output -->
+  <navindex>
+    <tab type="mainpage" visible="yes" title="About"/>
+    <tab type="usergroup" url="BEGIN_URLInstallationGuideEND_URL" title="Getting Started">
```

Comment:
mb embrace BEGIN_URL -> $(BEGIN_URL)
mb embrace END_URL -> $(END_URL)

Reply:
$(BEGIN_URL) - looks like call get function result in bash
i use __x__